config/jobs/kubernetes/publishing-bot/publishing-bot-presubmits.yaml

```diff
@@ -9,3 +9,25 @@ presubmits:
       - image: golang:1.11
         command:
         - make
+  kubernetes/kubernetes:
+    - name: pull-publishing-bot-build
+      always_run: false
+      skip_report: false
+      optional: true
+      run_if_changed: '^staging/publishing.*$'
+      decorate: true
+      path_alias: k8s.io/kubernetes
+      spec:
+        containers:
+          - image: gcr.io/k8s-testimages/kubekins-e2e:v20190205-d83780367-master
+            args:
+              - --repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)
+              - --repo=k8s.io/publishing-bot=master
+              - --root=/go/src
+              - --upload=gs://kubernetes-jenkins/logs
+              - --scenario=execute
+              - --
```

Comment:
Since you have `decorate: true` 👍, every arg before and including this -- can be removed and s/args/command.

Then add an extra_refs entry for the publishing bot repo

Reply:
On the job config (not podspec)
```yaml
extra_refs:
 - org: kubernetes
   repo: publishing-bot
   base_ref: master
   path_alias: k8s.io/publishing-bot
```

Reply:
fixing!

Reply:
thanks!